formats/opentype.fathom

```diff
@@ -823,6 +822,50 @@ let htmx_table = fun (number_of_long_horizontal_metrics : U16) => fun (num_glyph
 // - [Microsoft's OpenType Spec: maxp — Maximum Profile](https://docs.microsoft.com/en-us/typography/opentype/spec/maxp)
 // - [Apple's TrueType Reference Manual: The `'maxp'` table](https://developer.apple.com/fonts/TrueType-Reference-Manual/RM06/Chap6maxp.html)
 
+/// Fields specific to maxp version 1.0
+let maxp_version_1 = {
+    /// Maximum points in non-composite glyphs.
+    max_points <- u16be,
+    /// Maximum contours in non-composite glyphs.
+    max_contours <- u16be,
+    /// Maximum points in composite glyphs.
+    max_composite_points <- u16be,
+    /// Maximum contours in composite glyphs.
+    max_composite_contours <- u16be,
+    /// Should be set to `2` in most cases.
+    ///
+    /// | Value | Meaning                                           |
+    /// | ----- | ------------------------------------------------- |
+    /// | `1`   | instructions do not use the twilight zone (Z0)    |
+    /// | `2`   | instructions do use Z0                            |
+    max_zones <- u16be,
+    /// Maximum points used in in the twilight zone (Z0).
+    max_twilight_points <- u16be,
+    /// Number of Storage Area locations.
+    max_storage <- u16be,
+    /// Number of function definitions (FDEFs).
+    max_function_defs <- u16be,
+    /// Number of instruction definitions (IDEFs).
+    max_instruction_defs <- u16be,
+    /// Maximum stack depth across the Font Program ('fpgm' table), the
+    /// Control Value Program ('prep' table), and all glyph instructions
+    /// (in the 'glyf' table)
+    max_stack_elements <- u16be,
+    /// Maximum size in bytes used for all glyph instructions.
+    max_size_of_instructions <- u16be,
+    /// Maximum number of components referenced at “top level” of all
+    /// composite glyphs.
+    max_component_elements <- u16be,
+    /// Maximum levels of recursion used when constructing compound glyphs.
+    ///
+    /// | Value | Meaning                                           |
+    /// | ----- | ------------------------------------------------- |
+    /// | `0`   | the font only contains simple glyphs              |
+    /// | `1`   | compound glyphs only contain simple glyphs (there are no components within components) |
+    max_component_depth <- { depth <- u16be | u16_lte depth 16 }, // where max_component_depth in 0..16
```

Comment:
We really do need some sugar for this haha 😅

Reply:
Sugar has been added in #370! 🍭

Reply:
Oops I forgot about this PR. I'll give it a touch up now.